chaoscf/actions.py

```diff
@@ -29,6 +29,22 @@ def delete_app(app_name: str, configuration: Configuration, secrets: Secrets,
     call_api(path, configuration, secrets, method="DELETE")
 
 
+def start_app(app_name: str, configuration: Configuration, secrets: Secrets,
+              org_name: str = None, space_name: str = None):
+    """
+    Start application
+
+    See https://apidocs.cloudfoundry.org/280/apps/updating_an_app.html
+    """
+    app = get_app_by_name(
+        app_name, configuration, secrets, org_name=org_name,
+        space_name=space_name)
+
+    path = "/v2/apps/{a}".format(a=app['metadata']['guid'])
+    call_api(
```

Comment:
Can we return the call_api() returned value too? It may be handy in the logs :)

Reply:
Right now other actions also call "call_api" without returning the value, and I am trying to follow the conventions to make the actions behavior consistent.

Again if I only return call_api in "start_app" action, it would break the action/probe behavior consistency. If we would like to change all the actions behavior, the change is then bigger than
the current pull request(which only adds an action), and would probably be better handled as a separate code push.